client/src/lib/helper.js

```diff
@@ -1,22 +1,43 @@
 import $ from 'jquery';
 
-const BASE_URL = 'http://localhost:3001';
-
 const getReservationInfo = (id, date, callback) => {
   $.ajax({
-    url: `${BASE_URL}/restaurants/${id}/reservations/${date}`,
```

Comment:
did putting the base url inside this property cause any problems?

Reply:
It worked fine but I ended up taking out the BASE_URL because I don't think it's good practice if the client is being served from the same server (a hassle if you change the port, etc.)